### ISSUE TITLE
Error upon updating my projects

### ISSUE THREAD

**Issue by dotinspace** [nyyManni/ejira#13]
Error when trying to update my issues with `ejira-update-my-projects`:

```
Request failed: invalid request: ((errorMessages Error in JQL Query: Expecting either a value, list of function but got ')'. You must surround ')' in quotation mark to use it as a value. (line 1, character 30)) (errors))
```

**Comment by nyyManni** [comment]
Seems that jql cannot handle an empty list with `IN` operator.
I pushed a change to `ejira-update-project` so that it does not attempt a request if the number of issues affected is zero.

The way `ejira-update-my-projects` does not work as well as I would like it to work, mostly because after a ticket is resolved through the web interface it is quite painful to get that state to sync into ejira. What I have started doing is instead of using `ejira-update-my-projects`, I use my tickets through org-agenda with the `ejira-jql` agenda filter. This way those tickets just stop appearing in my agenda after they have been marked as resolved through the web interface.

Ideas to improve on the method of updating the tickets via `ejira-update-my-projects` are greatly welcome (not everyone is using boards), there is ticket #11 I created to follow thoughts on that.
